### PR TITLE
Add unit test for timeTrackingTable

### PR DESCRIPTION
**Pull request by vick3d**
#### Description
As a development team,
In order to test for react components,
We would like to setup unit tests for our timeTrackingTable

Changes proposed in this pull request:
* add unit tests
* mock response and alerts
#### What I have learned working on this feature: 
To mock out resolve responses and alerts

### CODE REVIEW

File: src/components/timeTrackingTable.js [hunk 1]
```diff
@@ -1,61 +1,62 @@
...
 			activity: this.state.activity,
 			description: "description",
 			fixedRate: "0.0",
 			hourlyRate: this.state.hourlyRate
-		}
-    try {
-			await saveData(values).then( response => {
-				if (response.message === "Entry saved"){
+		};
+		console.log(values)
```
Comment (perhegg) [inline src/components/timeTrackingTable.js]: remove this

Comment (vick3d) [reply src/components/timeTrackingTable.js]: done :)

File: src/components/timeTrackingTable.js [hunk 2]
```diff
@@ -1,61 +1,61 @@
...
+				if (response.message === "Entry saved") {
 					this.entryHandler();
 				} else {
-					alert(response.message)
+					alert(response.message);
 				}
 			});
-    } catch(error) {
+		} catch (error) {
 			console.log(error);
```
Comment (LauraRe) [inline src/components/timeTrackingTable.js]: do we need this?

Comment (vick3d) [reply src/components/timeTrackingTable.js]: yes, we need to catch if the try function return an error.

**Pull request merged**